bin/aws_cleaner.rb

```diff
@@ -147,14 +152,29 @@ def closelog(message)
       instance_id = AwsCleaner.new.process_message(body)
 
       if instance_id
-        chef_node = AwsCleaner::Chef.get_chef_node_name(instance_id, @config)
-        webhook(id, instance_id)
-        chef(id, instance_id, chef_node)
-        sensu(id, instance_id, chef_node)
+        node = case @config[:lookup]
```

Comment:
This option name seems too ambiguous to me, short variables are good but maybe making it a bit more verbose will do some good here. maybe something along the lines of `lookup_type`?

Reply:
Have updated it to lookup_type. That is a bit more clear.